scripts/sct_create_mask.py

```diff
@@ -134,7 +133,7 @@ def create_mask(param):
         sct.run(['sct_image', '-i', 'point.nii.gz', '-o', 'point_RPI.nii.gz', '-setorient', 'RPI', '-v', '0'], verbose=False)
     #
     # if method_type == 'centerline':
-    #     orientation_centerline = get_orientation_3d(method_val, filename=True)
+    #     orientation_centerline = Image(method_val).orientation
```

Comment:
that's commented-out code, so obviously no impact, but if it were uncommented...
orientation has been set because loadFromPath has set it -> no problem